### PR TITLE
Add a pr template and modify the contributor guidelines

### PR DESCRIPTION
**Pull request by ibrand**
Very open to feedback on this, but here is an initial pass at a pr_template for the repo

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+- UI Change (change that will effect the visual experience)
+
+# Checklist:
+- I have manually tested my changes on desktop and mobile
```
Comment (shakalee14) [inline pull_request_template.md]: should this portion be an actual checklist? so `- [ ] `

Comment (ibrand) [reply pull_request_template.md]: Oh it totally should. It was rendering as a checkbox in my markdown editor for some reason. Thanks for seeing that!

**Comment by ibrand** [comment]
Made a few changes based on actually using the template to make it a little shorter and easier to fill out while still staying rigorous with the asks

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change, and any blockers that make your change a WIP*
+
+# Checklist:
+- [ ] I have manually tested my changes on desktop and mobile
+- [ ] The test suite passes locally with my changes
+- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
+- [ ] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
+- [ ] I have made any necessary changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] My PR is labeled [WIP] if it is in progress
```
Comment (shakalee14) [inline pull_request_template.md:12]: can WIP be in the title? or literally adding the label wip - just want to make sure this is clear. can everyone add labels at this level?

Comment (ibrand) [reply pull_request_template.md:12]: Oh yeah! I was imagining this being in the title. I can make that more clear since some folks can't add labels

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**